packages/palette/src/elements/Pagination/LargePagination.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { color } from "../../helpers/color"
+import { ChevronIcon } from "../../svgs/ChevronIcon"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Sans } from "../Typography"
+
+export interface Props {
+  onClick?: (cursor: string, page: number) => void
```

Comment:
I'm surprised we're encoding knowledge about cursors, etc into what should be a display component (I realize this was already here)

Reply:
There's been some back and forth about this (and I agree). This pattern is used across a couple codebases due to how we've designed our backend pagination cursor implentation, with `cursor` being used as part of the language. We could say "page" or whatever. 

Once we have a design for handling actual links, this can be refactored out, but it will also impact volt.